openedx/core/djangoapps/user_authn/views/login.py

```diff
@@ -360,7 +360,8 @@ def _track_user_login(user, request):
         {
             'category': "conversion",
             'label': request.POST.get('course_id'),
-            'provider': None
+            'provider': None,
+            'register_intent': bool(request.POST.get('register_intent')),
```

Comment:
It would be always true if there would be any value in `register_intent` attribute.
Like:
```
bool('False')
--> True
```

Reply:
its changed